rules/codelingo/go/reassigned-error/codelingo.yaml

```diff
@@ -14,8 +14,45 @@ funcs:
 tenets:
 - name: reassigned-error
   actions:
+      codelingo/docs:
+        title: Check `err` Before Variable Re-Assignment
+        body: | 
+          When re-assigning a variable in Go, it is convention to check the `err` prior to
```

Comment:
It's not so much convention as _correctness_ – errors contain important information, and if you're throwing away that information without reacting to it, you're probably making a mistake.

Reassigning _variables in general_ is not generally a problem; reassigning _error variables_ almost always is.